sap-successfactors/4.0/modules/ROOT/pages/sap-successfactors-connector-examples.adoc

```diff
@@ -1,161 +1,538 @@
-
-Template for Including Multiple Examples in One Examples Topic
-
-// Use this template under one of these circumstances to include multiple examples in the 
-// Examples topic:
-
-// The examples are related. 
-// There are only a few examples and the documentation for each one is fairly short.
-
-////
-For an example of this template, see NetSuite Connector 10.0 Examples - Mule 4. Consider using the Template for Including Multiple Examples in One Examples Topic if you have multiple, unrelated examples.
-////
-
-= <connector> Examples - Mule 4
-
-The following examples show how to <action> with <connector-name>: 
-
-<Hyperlink 1>
-<short description of the example>
-<Hyperlink 2>
-<short description of the example>
-.
-.
-.
-These examples use variables for some field values. You can either:
-Replace the variables with their values in the code
-Provide the values for each variable in a properties file and then refer to that file from the connector configuration.
-If you don’t know how to use a properties file, see Configuring Property Placeholders.
-
-== <Example 1>
-
-// Use subsections, if needed, to show configuration steps. 
-=== Subsection 1 
-=== Subsection 2
-=== Subsection 3
-.
-.
-.
-// Add these steps to the end of the numbered list:
-. Save the project. 
-. Test the app by sending a <REST command> request to <url>.
-
-=== XML for <what the example shows>
-
-Paste this code into the Studio XML editor to quickly load the flow for this example into your Mule app:
-
-// <Copy the XML from Studio and paste it here. 
-// Remove the  attributes from the XML before including it in the topic.>
-
-// If you aren’t splitting the example into subsections, add these subsections instead:
-
-=== Steps for <what the example shows>
-
-// Add these steps to the end of the numbered list:
-. Save the project. 
-. Test the app by sending a <REST command> request to <url>.
-
-=== XML for <what the example shows>
-
-Paste this code into your Studio XML editor to quickly load the flow for this example into your Mule app:
-
-// <Copy the XML from Studio and paste it here. 
-// Remove the  attributes from the XML before including it in the topic.>
-
-== <Example 2>
-
-// Same template for Example 1
-
-== <Example 3>
-.
-.
-.
+These examples show how to create flows to invoke some of SuccessFactor's standard operations, including how to:
+
+* <<create-query-flow,Create a query to query across entities>>
+* <<create-entry,Create an entry on SuccessFactors>>
+* <<get-entity-by-id,Get an entity by ID>>
+* <<update-entry,Create and update an entry on SuccessFactors>>
+* <<delete-entry, Delete an entry from SuccessFactors>>
+
+== Prerequisites
+
+To build and run these flows, you need:
+
+* OpenJDK 8 or 11
+* Anypoint Studio 7.1 or later
+* Mule 4.2.2 or later
+* SuccessFactors Connector 3.x or later
+* SuccessFactors credentials to send OData API requests
+* DataWeave 2.0
+
+=== Prerequisite Steps
```

Comment:
These are not all prerequisites.

Reply:
They are all prerequisite steps actually. They need to do this before they move on to each example